ledfx/effects/plasmawled.py

```diff
@@ -124,7 +123,9 @@ def draw(self):
             self.time += self.speedb
             time_val = int(self.time * 1000)
         else:
-            time_val = int((timeit.default_timer() - self.start_time) * 1000)
+            time_val = int(
+                (self.current_time - self.EFFECT_START_TIME) * 1000
+            )  # TODO: Confirm this logic/timer structure
```

Comment:
I may have broken this one -  I'm not sure I grok it, from what I could see this was reaaaaly small changes of frametime.

However I might have broke this with using EFFECT_START_TIME.

Dunno hahaha

Reply:
I will look at this one in the same context as the plasma2d question.

Reply:
I wonder if this one can drop to just self.current_time, everything likely loops and gets rid of another use of self.EFFECT_START_TIME

Reply:
Yeah I couldn't see a difference - removed.